postgres-driver/sqlc/schema.sql

```diff
@@ -1,108 +1,112 @@
-    success_rate float NOT NULL,
-	created_at timestamp     NOT NULL  ,
-	updated_at timestamp     NOT NULL  ,
-	CONSTRAINT pk_service_record PRIMARY KEY ( id , portal_region_name )
+	id bigint NOT NULL GENERATED ALWAYS AS IDENTITY,
+	node_public_key char(64) NOT NULL,
+	pokt_chain_id char(4) NOT NULL,
+	session_key char(44) NOT NULL,
+	request_id varchar NOT NULL,
+	portal_region_name varchar NOT NULL,
+	latency float NOT NULL,
+	tickets integer NOT NULL,
+	result varchar NOT NULL,
+	available boolean NOT NULL,
+	successes integer NOT NULL,
+	failures integer NOT NULL,
+	p90_success_latency float NOT NULL,
+	median_success_latency float NOT NULL,
+	weighted_success_latency float NOT NULL,
+	success_rate float NOT NULL,
+	created_at timestamp NOT NULL,
+	updated_at timestamp NOT NULL,
+	CONSTRAINT pk_service_record PRIMARY KEY (id, portal_region_name)
 );
-
-ALTER TABLE relay ADD CONSTRAINT fk_relay_portal_region FOREIGN KEY ( portal_region_name ) REFERENCES portal_region( portal_region_name );
-
-ALTER TABLE relay ADD CONSTRAINT fk_relay_session FOREIGN KEY ( session_key ) REFERENCES pocket_session( session_key );
-
-ALTER TABLE service_record ADD CONSTRAINT fk_service_region_portal_region FOREIGN KEY ( portal_region_name ) REFERENCES portal_region( portal_region_name );
-
-ALTER TABLE service_record ADD CONSTRAINT fk_service_record_session FOREIGN KEY ( session_key ) REFERENCES pocket_session( session_key );
-
-ALTER TABLE pocket_session ADD CONSTRAINT fk_pocket_session_portal_region FOREIGN KEY ( portal_region_name ) REFERENCES portal_region( portal_region_name );
-
+ALTER TABLE relay
+ADD CONSTRAINT fk_relay_portal_region FOREIGN KEY (portal_region_name) REFERENCES portal_region(portal_region_name);
+ALTER TABLE relay
+ADD CONSTRAINT fk_relay_session FOREIGN KEY (session_key) REFERENCES pocket_session(session_key);
+ALTER TABLE service_record
+ADD CONSTRAINT fk_service_region_portal_region FOREIGN KEY (portal_region_name) REFERENCES portal_region(portal_region_name);
+ALTER TABLE service_record
+ADD CONSTRAINT fk_service_record_session FOREIGN KEY (session_key) REFERENCES pocket_session(session_key);
+ALTER TABLE pocket_session
+ADD CONSTRAINT fk_pocket_session_portal_region FOREIGN KEY (portal_region_name) REFERENCES portal_region(portal_region_name);
 INSERT INTO portal_region (portal_region_name)
 VALUES ('europe-west3'),
-       ('europe-north1'),
-       ('europe-west8'),
-       ('europe-southwest1'),
-       ('europe-west2'),
-	   ('europe-west9'),
-	   ('us-east4'),
-	   ('us-east5'),
-	   ('us-west2'),
-	   ('us-west1'),
-	   ('northamerica-northeast2'),
-	   ('asia-east2'),
-	   ('asia-northeast1'),
-	   ('asia-northeast3'),
-	   ('asia-south1'),
-	   ('asia-southeast1'),
-	   ('australia-southeast1');
-
-INSERT INTO pocket_session
-(session_key, session_height, portal_region_name, created_at, updated_at)
-VALUES
-('', 1, 'europe-west3', TIMESTAMP '1970-01-01 00:00:00', TIMESTAMP '1970-01-01 00:00:00');
+	('europe-north1'),
```

Comment:
Not a blocker, and I understand this is just formatting existing code, but these should eventually be moved out of the code.